.github/workflows/travis.yml

```diff
@@ -0,0 +1,80 @@
+name: Tests ported from Travis
+
+on: [push]
+
+jobs:
+  build-linux:
+    if: "!startsWith(github.ref, 'refs/tags/') && !contains(github.event.head_commit.message, '[skip ci]')"
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 5
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+        pip: ["pip", setuptools ~= 42.0", "setuptools_scm[toml] >= 3.4"]
```

Comment:
You never use `${{ matrix.python-version }}` or `${{ matrix.pip }}`...

Reply:
github actions expands them automatically. Although, I don't think I'll use the pip partition after all.

I'll move to working on PRs, now that I github should enable actions for them.